One of the tests expects GNU strip.

This should be reported to upstream.

--- automake-1.17/t/strip3.sh
+++ automake-1.17/t/strip3.sh
@@ -41,7 +41,7 @@ $ACLOCAL
 $AUTOCONF
 $AUTOMAKE -a
 
-./configure --prefix="$(pwd)/inst" STRIP='strip --verbose'
+./configure --prefix="$(pwd)/inst" STRIP='gstrip --verbose'
 $MAKE
 $MAKE install-strip
 
